src/themes/default/index.js

```diff
@@ -17,6 +18,7 @@ function initTheme (app, router, store, config, ssrContext) {
   // You can do it on your own and then be able to customize the components used for example for German storeView checkout
   // To do so please execlude the desired storeView from the config.storeViews.mapStoreUrlsFor and map the urls by Your own like:
   // { name: 'de-checkout', path: '/checkout', component: CheckoutCustomized },
+  store.registerModule('themeCart', cartModule)
```

Comment:
you can also just extend `ui-store` (use `extendModule` with module that has `ui` key)

Reply:
@filrak Should I do this inside of `src/modules/ui-store` (like these comments says in this file)? In that case I'd have to import store configuration from `themes/default/store`, is that good idea? Or maybe should I create a new module (eg. `ui-store-cart`) with extending ui-store and move there all of store configuration that currently I have created in the `themes/default/store`?